manifests/e/ElectronicTeam/Flexihub/5.3.14268/ElectronicTeam.Flexihub.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ElectronicTeam.Flexihub
+PackageVersion: 5.3.14268

```

Comment:
```suggestion
PackageVersion: 4.0.12820
```

Reply:
Existing package version is correct file reporting wrong.

See: 
https://help.electronic.us/support/solutions/articles/44000768915-what-s-new-in-this-version-on-windows

On install:
https://1drv.ms/u/s!AmUD8H7LTq5351g4jGsTE-IdxttV?e=i16nQf

Reply:
The PackageVersion is unacceptable because it will cause a upgrade loop. The Publisher needs to fix their version.
Since they have not noticed this since it broke after 2020-08-19 good luck with that.